test/Msbuild.Tests.Utilities/Msbuild.Tests.Utilities.csproj

```diff
@@ -20,6 +20,9 @@
     <ProjectReference Include="..\..\src\Microsoft.DotNet.ProjectJsonMigration\Microsoft.DotNet.ProjectJsonMigration.csproj">
       <FromP2P>true</FromP2P>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Microsoft.DotNet.Cli.Sln.Internal\Microsoft.DotNet.Cli.Sln.Internal.csproj">
+      <FromP2P>true</FromP2P>
```

Comment:
FromP2P is used to indicate that this was added by migration because of transitive referencing. You don't need it when adding a real, direct dependency.